all-faq.md

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
What is the purpose of this page? Display all FAQs? Is it linked somewhere? I did not find any